content added (#2)
jcolomb/gintonic-blog: text/surveytext.md

Questions:

Intro.

> We have developed a folder structure template for research project, and are eager to get the feedback of researchers of different disciplines, in order to optimize it. This version of the template was created with neuroscience projects in mind, and designed to be used in combination with a git-based data management platform, like GIN (g-node infrastructure). Its use would thrive an homogenization of file organization, which we think will ease data management and facilitate research collaboration. Please have a look at the template and respond to this few questions.

Reviewer: ```suggestion
We have developed a folder structure template for research projects, and are eager to get the feedback of researchers of different disciplines, in order to optimize it. This version of the template was created with neuroscience projects in mind, and designed to be used in combination with a git-based data management platform, like GIN (g-node infrastructure) or GitLab. Its use would thrive an homogenization of file organization, which we think will ease data management and facilitate research collaboration. Please have a look at the template and respond to this few questions.
```

Please refer to BLOGPOST-LINK for more information.

![Fig.4: Folder structure developed after analyzing the survey. The template works mostly on the project level (one unique folder for all files related to one project). The experiment level is taken care by specifying several experiment sub-folders when new experiments are started. By sharing specific subfolder independently in a cloud solution, one can reorganize information in cross-project directories that host sub-folders coming from different projects. Note that both the creation of experiment sub-folders and the creation of laboratory level organization is automated in GIN-tonic.](../50_figures/pulication_ready_figures/fig4.png)

> 1. What is your research field ?

Reviewer: ```suggestion
1. What is your research field?
```

   - Life sciences
   - Other STEM (science, technology, engineering and maths)
   - SSH (Social science, art and humanities)

> 

Reviewer: I think we should also give the researcher the opportunity to specify his/her research field more concretely, e.g. by an additional comment for each option.

> 2. Would you consider using this template structure for your projects ?

Reviewer: ```suggestion
2. Would you consider using this template structure for your projects?
```

   - yes
   - yes but would modify some elements (please specify in the next question)
   - no
    

3. Do you have any suggestions for improving the current template structure?

- COMMENT BOX

> 4. Will you try to implement this template structure with your collaborators?

Reviewer: Why is this question so important? I think it is hard to answer with '... I will force it as much as I can' or '... try to convince them' if the template structure is quite new for me... Maybe here we have a risk to get answers which we do not really want to get... So: I would remove this question.

Author: the goal is to set who we should try to convince first, I will try to rephrase.
